timelock-api/src/main/conjure/cross-client-batched-timelock-api.yml

```diff
@@ -0,0 +1,63 @@
+types:
+  imports:
+    LeaderTime:
+      base-type: any
+      external:
+        java: com.palantir.lock.v2.LeaderTime
+    Long:
+      base-type: any
+      external:
+        java: java.lang.Long
+    LockWatchStateUpdate:
+      base-type: any
+      external:
+        java: com.palantir.lock.watch.LockWatchStateUpdate
+    ConjureIdentifiedVersion:
+      base-type: any
+      external:
+        java: com.palantir.atlasdb.timelock.api.ConjureIdentifiedVersion
```

Comment:
I don't know what opinions are on just adding these endpoints / requests to the current timelock api, or creating a new one (as you have done here) - but either way, I think it's really weird to import in definitions from the other API.

Reply:
I'd prefer we just use the existing API file. In my mind the model was that they would be placed on a different _service_, but in the same API.